runtime/src/weights/frame_system.rs

```diff
@@ -32,7 +32,7 @@ use frame_support::{
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::weights::WeightInfo for WeightInfo<T> {
     fn remark(_b: u32) -> Weight {
-        (8_454_000 as Weight)
+        (0 as Weight)
```

Comment:
This is strange, it should not be `0`. I'll revert this to the previous value, since `remark` did not change a bit.
```suggestion
        (8_454_000 as Weight)
```